addons/godot-xr-tools/effects/fade.gdshader

```diff
@@ -1,13 +1,13 @@
 shader_type spatial;
 render_mode depth_test_disabled, skip_vertex_transform, unshaded, cull_disabled;
 
-uniform float alpha = 0.0;
+uniform vec4 albedo : source_color;
 
 void vertex() {
 	POSITION = vec4(VERTEX.x, -VERTEX.y, 0.0, 1.0);
```

Comment:
Something we need to change here that I didn't realise until later is that this needs to be `POSITION = vec4(VERTEX.x, -VERTEX.y, -1.0, 1.0);` when using OpenGL (compatibility). The code as it stands right now is correct for Vulkan.

Once the reverse-Z PR is merged in 4.3, we need to change this again to `POSITION = vec4(VERTEX.x, -VERTEX.y, 1.0, 1.0);`

Reply:
This shader works on the current options (tested on PC, not native):
- Godot 4.1.3 Vulkan & OpenGL
- Godot 4.2.1 Vulkan & OpenGL
- Godot 4.3-dev5 Vulkan & OpenGL

Back in 4.0 we were looking for an option that would satisfy both OpenGL and Vulkan, and I thought the ``cull_disabled` was all we needed.

Reply:
What would we need to do if we want this shader to work in Vulkan & OpenGL on Godot 4.1 - 4.3?

In the Reverse Z Progress Report announcement there didn't seem to be anything about how to handle forwards/backwards compatibility.